### PR TITLE
add: `davinci-resolve`

### PR DESCRIPTION
**Pull request by vigress8**

### CODE REVIEW

**Comment by vigress8** [comment]
Package seems to be too large for the runners. It works on my machine

**Comment by oklopfer** [comment]
I need elsie to review this one bc of its sussiness

**Comment by Elsie19** [comment]
> I need elsie to review this one bc of its sussiness

Ok.

File: packages/davinci-resolve/davinci-resolve.pacscript [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+  reqjson=$(
+    {
+      IFS= read -rd ''
+      # shellcheck disable=SC2086
+      echo ${REPLY}
+    } <<- JSON
+	{
+	  "firstname": "Pacstall",
+	  "lastname": "Linux",
+	  "email": "someone@pacstall.dev",
```
Comment (Elsie19) [inline packages/davinci-resolve/davinci-resolve.pacscript:236]:
```suggestion
	  "email": "pacstall@pm.me",
```

**Review by Elsie19 — CHANGES_REQUESTED** [review]

**Review by Elsie19 — APPROVED** [review]

**Pull request merged**